pkg/codegen/docs/gen_creation_example_for_csharp.go

```diff
@@ -0,0 +1,269 @@
+package docs
```

Comment:
Very minor nit - I don't think we need `_for` in the name of these files.

Reply:
Fixed ✅